sympy/utilities/pytest.py

```diff
@@ -5,6 +5,7 @@
 import sys
 import functools
 import os
+import re
 
 from sympy.core.compatibility import get_function_name
 
```

Comment:
You should consider `py.test` compatibility.

Reply:
@ylemkimon can you brief me more about what compatibility of `pytest` means?

Reply:
SymPy tests should be compatible with [`pytest`](https://docs.pytest.org/en/latest/), however if you add `assert_raise_message` there, it'll not be available when running tests using `pytest`. Also if you modify existing API, it should be a subset of pytest's API.